src/app/filters/examples/filter-type-ahead-example.component.html

```diff
@@ -0,0 +1,70 @@
+<div class="padding-15">
+  <div class="row padding-bottom-15">
+    <div class="col-xs-12">
+      <h4>Filter Component Example</h4>
+      <hr/>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-sm-12">
+      <div class="form-group">
+        <pfng-filter [config]="filterConfig"
+                     (onChange)="filterChanged($event)"
+                     (onFieldSelect)="filterFieldSelected()"
+                     (onTypeAhead)="filterQueries($event)"></pfng-filter>
+      </div>
+    </div>
+  </div>
+  <div class="row padding-bottom-15 padding-top-15">
+    <div class="col-xs-12">
+      <h4>Valid Items:</h4>
+      <hr/>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-12">
+      <div *ngFor="let item of items" class="col-md-12">
+        <div class="row">
+          <div class="col-md-3">
+            <span>{{item.name}}</span>
+          </div>
+          <div class="col-md-5">
+            <span>{{item.address}}</span>
+          </div>
+          <div class="col-md-2">
+            <span>{{item.birthMonth}}</span>
+          </div>
+          <div class="col-md-2">
+            <span>{{item.weekDay}}</span>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-12">
+      <label class="events-label">Current Filters: </label>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-12">
+      <textarea rows="5" class="col-md-12">{{filtersText}}</textarea>
+    </div>
+  </div>
+  <div class="row padding-bottom-15 padding-top-15">
+    <div class="col-xs-12">
+      <h4>Code</h4>
+      <hr/>
+    </div>
+  </div>
+  <div>
+    <tabset style="border: 1px black;">
```

Comment:
This is just the example code. I'm planning to replace that with a style class (for all examples) after all the latest PRs are merged -- it's been merge h*ll getting everything integrated.